src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -424,8 +424,9 @@ private X509Crl CreateCrl(X509Certificate signingCertificate)
             
             _crlGenerator.Reset();
 
```

Comment:
Maybe outside the scope of this PR, but having a single _crlGenerator that is reset and mutated makes me extremely nervous.  It raises the possibility of multiple tests intefering with each other.  I would much rather see it be rewritten so it is not necessary to call Reset or set properties like this.  Better to create some kind of object containing all the parameters that gets passed to the _crlGenerator.